octoprint_mrbeam/support.py

```diff
@@ -13,6 +13,8 @@
 
 _logger = mrb_logger("octoprint.plugins.mrbeam.support")
 
+USB_MOUNT_DIR = "/media/"
+
```

Comment:
Remove `SUPPORT_STICK_FILE_MAX_AGE`

Reply:
well it didn't show in the refered lines here, but you should delete the definition